### DISCUSSION TITLE
Allow collapsible with default group

### DISCUSSION THREAD

**Discussion by SimonMacIntyre** [filamentphp/filament#8802] [Ideas]
Example: 
```php
 ->defaultGroup('topicGroup.title')->collapsible(),
```
The use case: I don't want to use the Group format, because it adds a dropdown in the header to choose groups, but I will only ever allow this specific single group. 

I like that `default group` does not add the header group toggle. However, I cannot add a `collapsible()` in this format, which is only possible by making it in the `Group` format, which forces the additional header filter dropdown from what I can tell.

**Comment by Racterrus** [comment]
You can! Try this:
`->defaultGroup(
                Group::make('parent.name')->collapsible(),
            )`
